ui/admin/app/components/credential-stores/credential-store/actions/index.hbs

```diff
@@ -2,37 +2,26 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: BUSL-1.1
 }}
-
-<Rose::Dropdown
-  @text={{t 'actions.manage'}}
-  @dropdownRight={{true}}
-  as |dropdown|
->
+<Hds::Dropdown data-test-manage-credential-stores as |dd|>
```

Comment:
```suggestion
<Hds::Dropdown data-test-manage-credential-stores-dropdown as |dd|>
```
change to match naming convention in other manage dropdowns

Reply:
Thanks for the catch Lis!